ui/app/components/transit-key-actions.js

```diff
@@ -65,37 +65,40 @@ export default Component.extend(TRANSIT_PARAMS, {
   onRefresh() {},
   init() {
     this._super(...arguments);
+
+    // eslint-disable-next-line ember/no-get
     if (get(this, 'selectedAction')) {
```

Comment:
We should probably figure out why this needs the `get` at some point

Reply:
I'll go ahead and add a comment.